setup.cfg

```diff
@@ -15,28 +15,31 @@ classifiers =
 packages = find:
 python_requires = >=3.8
 install_requires =
-    certifi >= 14.5.14
-    frozendict ~= 2.3.4
-    setuptools >= 21.0.0
-    typing-extensions ~= 4.3.0
-    urllib3 ~= 1.26.7
-    python-dateutil
-    absl-py
-    tqdm
-    jinja2
-    numpy
-    scipy
-    matplotlib
-    xarray
-    vtk
-    imageio
-    pyvista
-    imageio-ffmpeg
-    ipython
-    nglview
-    MDAnalysis
-    utm
-    dask
+  certifi >= 14.5.14
+  frozendict ~= 2.3.4
+  setuptools >= 21.0.0
+  typing-extensions ~= 4.3.0
+  urllib3 ~= 1.26.7
+  python-dateutil
+  absl-py
+  tqdm
+  jinja2
+  numpy
+  scipy
+  matplotlib
+  xarray
+  vtk
+  imageio
+  pyvista
+  imageio-ffmpeg
```

Comment:
isn't this a highly specific package that requires external libs?

Reply:
Yes, but since it is related with molecules, it'll be done in a different PR